src/java/org/apache/cassandra/tools/nodetool/stats/TableStatsHolder.java

```diff
@@ -275,6 +284,66 @@ private void initializeKeyspaces(NodeProbe probe, boolean ignore, List<String> t
                     }
                 }
 
+                double[] ucsSSTableTokenSpace = table.getPerLevelAvgTokenSpace();
+                if (ucsSSTableTokenSpace != null)
+                {
+                    statsTable.isUCSSstable = true;
+                    for (int level = 0; level < ucsSSTableTokenSpace.length; level++)
+                    {
+                        statsTable.sstableAvgTokenSpaceInEachLevel.add(String.format("%.03f", ucsSSTableTokenSpace[level]));
+                    }
+                }
+
+                double[] ucsMaxDensityThreshold = table.getPerLevelMaxDensityThreshold();
+                if (ucsMaxDensityThreshold != null)
+                {
+                    statsTable.isUCSSstable = true;
+                    for (int level = 0; level < ucsMaxDensityThreshold.length; level++)
+                    {
+                        statsTable.sstableMaxDensityThresholdInEachLevel.add(String.format("%.03f", ucsMaxDensityThreshold[level]));
+                    }
+                }
+
+                double[] ucsSsTableAvgSize = table.getPerLevelAvgSize();
```

Comment:
```suggestion
                double[] ucsSSTableAvgSize = table.getPerLevelAvgSize();
```
Please make the variable name consistent with other variable names